packages/server/src/handlers/BaseHandler.ts

```diff
@@ -3,11 +3,19 @@ import EventEmitter from 'node:events'
 import type {ServerOptions} from '../types'
 import type {DataStore} from '../models'
 import type http from 'node:http'
+import stream from 'node:stream'
+import {ERRORS} from '../constants'
 
 const reExtractFileID = /([^/]+)\/?$/
 const reForwardedHost = /host="?([^";]+)/
 const reForwardedProto = /proto=(https?)/
 
+export interface CancellationContext {
```

Comment:
Can you please add documentation for this interface? What do the properties `abort` and `cancel` mean?

Reply:
Sure thing!

Reply:
Done